project/accounts/views.py

```diff
@@ -24,20 +23,41 @@ def form_valid(self, form):
         return redirect(self.success_url)
 
 
-@login_required
-def user_profile(request):
-    if request.method == "POST":
-        user_form = UpdateUserForm(request.POST, instance=request.user)
+# @login_required
+# def user_profile(request):
+#     if request.method == "POST":
+#         user_form = UpdateUserForm(request.POST, instance=request.user)
+
+#         if user_form.is_valid():
+#             user_form.save()
+
+#             messages.success(request, _("Profile updated successfully"))
+
+#             return redirect(to="user-profile")
+#         else:
+#             messages.error(request, _("Profile form isn't valid"))
+#     else:
+#         user_form = UpdateUserForm(instance=request.user)
```

Comment:
We can remove the commented code, since your CBV approach is much nicer :smile: 

It would also make things slightly easier to review.
```suggestion
```